src/commands/process_metavaults.py

```diff
@@ -0,0 +1,202 @@
+import asyncio
+import logging
+import sys
+from pathlib import Path
+
+import click
+from eth_typing import ChecksumAddress
+from eth_utils import to_checksum_address
+from sw_utils import InterruptHandler
+from web3.types import Gwei
+
+from src.commands.start.base import log_start, setup_sentry
+from src.common.checks import wait_execution_catch_up_consensus
+from src.common.clients import setup_clients
+from src.common.consensus import get_chain_finalized_head
+from src.common.logging import LOG_LEVELS, setup_logging
+from src.common.protocol_config import update_oracles_cache
+from src.common.utils import log_verbose
+from src.common.validators import validate_eth_addresses
+from src.config.networks import AVAILABLE_NETWORKS, GNOSIS, MAINNET, NETWORKS
+from src.config.settings import (
+    DEFAULT_META_VAULT_UPDATE_INTERVAL,
+    DEFAULT_MIN_DEPOSIT_AMOUNT_GWEI,
+    LOG_FORMATS,
+    LOG_PLAIN,
+    settings,
+)
+from src.meta_vault.tasks import ProcessMetaVaultTask
+
+logger = logging.getLogger(__name__)
+
+
+@click.option(
+    '--vaults',
+    callback=validate_eth_addresses,
+    envvar='VAULTS',
+    prompt='Enter the comma separated list of your meta vault addresses',
+    help='Addresses of the meta vaults to process.',
+)
+@click.option(
+    '--max-fee-per-gas-gwei',
+    type=int,
+    envvar='MAX_FEE_PER_GAS_GWEI',
+    help=f'Maximum fee per gas for transactions. '
+    f'Default is {NETWORKS[MAINNET].MAX_FEE_PER_GAS_GWEI} Gwei for Ethereum, '
+    f'{NETWORKS[GNOSIS].MAX_FEE_PER_GAS_GWEI} Gwei for Gnosis.',
+)
+@click.option(
+    '--wallet-password-file',
+    type=click.Path(exists=True, file_okay=True, dir_okay=False),
+    envvar='WALLET_PASSWORD_FILE',
+    help='Absolute path to the wallet password file. '
+    'Default is the file generated with "create-wallet" command.',
+)
+@click.option(
+    '--wallet-file',
+    type=click.Path(exists=True, file_okay=True, dir_okay=False),
+    envvar='WALLET_FILE',
+    help='Absolute path to the wallet. '
+    'Default is the file generated with "create-wallet" command.',
+)
+@click.option(
+    '-v',
+    '--verbose',
+    help='Enable debug mode. Default is false.',
+    envvar='VERBOSE',
+    is_flag=True,
+)
+@click.option(
+    '--meta-vault-min-deposit-amount-gwei',
+    type=int,
+    envvar='META_VAULT_MIN_DEPOSIT_AMOUNT',
+    help=f'Minimum amount in gwei to deposit into subvaults for meta vault.'
+    f' The default is {DEFAULT_MIN_DEPOSIT_AMOUNT_GWEI}',
+    default=DEFAULT_MIN_DEPOSIT_AMOUNT_GWEI,
+)
+@click.option(
+    '--meta-vault-update-interval',
+    type=int,
+    envvar='META_VAULT_UPDATE_INTERVAL',
+    help=f'Interval in seconds to process subvault states and deposits.'
+    f' The default is {DEFAULT_META_VAULT_UPDATE_INTERVAL}',
+    default=DEFAULT_META_VAULT_UPDATE_INTERVAL,
+)
```

Comment:
This parameter is not needed. We update state when it's possible and process deposits when META_VAULT_MIN_DEPOSIT_AMOUNT_GWEI has been accumulated